apps/console/src/features/core/constants/app-constants.ts

```diff
@@ -316,6 +317,35 @@ export class AppConstants {
             .set("ADMINISTRATORS", `${AppConstants.getAdminViewBasePath()}/administrators`);
     }
 
+    /**
+     * Filter governance connectors for the side panel for a sub organization.
+
+     * @param governanceConnectorCategories - List of governance connector categories to evaluate.
+     * 
+     * @returns Filtered governance connector categories.
+     */
+    public static filterGoverananceConnectors(
+        governanceConnectorCategories: GovernanceConnectorCategoryInterface[]
+    ) : GovernanceConnectorCategoryInterface[] {
+        const showGovernanceConnectorsIdOfSuborgs = [];
+
+        GovernanceConnectorUtils.SHOW_GOVERNANCE_CONNECTORS_FOR_SUBORGS
+            .forEach(connector => {
+                showGovernanceConnectorsIdOfSuborgs.push(connector.id);
+            });
+
+        for (let i = governanceConnectorCategories.length-1; i >=0 ; i--) {
```

Comment:
We can use inbuilt filter method here.

Reply:
Addressed with `https://github.com/wso2/identity-apps/pull/3928/commits/c5d3192904e44f0673b9003e00d9e0c9ca2c04f9`